src/components/Button/Button.css

```diff
@@ -65,12 +66,26 @@
   color: $color__river;
 }
 
-.y-button:disabled,
 .y-button__state-disabled {
   opacity: 0.5;
   cursor: default;
 }
 
+.y-button__state-loading {
+  cursor: default;
+}
+
+.y-button__state-loading .y-button--contents {
+  visibility: hidden;
+}
+
+.y-button__state-loading .y-button--spinner {
```

Comment:
this selector can just be `.y-button--spinner`

Reply:
True.